lib/taskana-core/src/main/java/pro/taskana/task/internal/models/TaskSummaryImpl.java

```diff
@@ -21,6 +21,8 @@
 /** Entity which contains the most important information about a Task. */
 public class TaskSummaryImpl implements TaskSummary {
 
+  static final int DEFAULT_MANUAL_PRIORITY = -1;
```

Comment:
Since this is only used during the initialization of `manualPriority` a constant is not really necessary.
You can either inline this or make it `private`